lib/hexpm/utils.ex

```diff
@@ -267,4 +267,45 @@ defmodule Hexpm.Utils do
     diff_url = Application.fetch_env!(:hexpm, :diff_url)
     "#{diff_url}/diff/#{package_name}/#{previous_version}..#{version}"
   end
+
+  @doc """
+  Returns a RFC 2822 format string from a UTC datetime.
+  """
```

Comment:
Let's add a note that we'll be able to simplify this code soon!

```suggestion
  """
  # TODO: Replace with `Calendar.strftime(datetime, "%a, %b %d %Y %H:%H:%S GMT")` on Elixir v1.11
```